code/script.js

```diff
@@ -29,19 +59,643 @@ const showMessage = (message, sender) => {
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
-
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1
+  showMessage(`Hello there Beat Bot!`, 'user')
+  setTimeout(() => botGreeting(), 1000)
+}
+
+//starts conversation with button
+welcomeButton.onclick = function () {
+  if (welcomeButton.style.display !== 'none') {
+    welcomeButton.style.display = 'none'; {
+      greeting(showMessage, 'user')
+    }
+  };
+}
+
+//Bot greets human
+const botGreeting = () => {
+  showMessage(`Hello there human, what's your name?`, 'bot')
+  questionNumber = 1
+}
+//user tells their name
+const handleNameInput = () => {
+  showMessage(`My name is ${input.value}`, 'user');
+  setTimeout(() => whatMood(), 1000);
+}
+
+//Pick your mood
+const whatMood = () => {
+  showMessage(`Oh ${input.value} describe your mood in one word!`, 'bot')
+  questionNumber = 3
+  input.value=''
+}
+//User choose their mood
+const moodAnswer = () => {
+  const inputMood = input.value
+  questionNumber = 4
+  showMessage(`I am ${inputMood}`, 'user')
+  setTimeout(() => songSelection(input.value), 1000);
+}
+
+//Selects a song from the list
+const songSelection = (input) => {
+  questionNumber = 5
+  
+//if sad, depressed or blue is selected these songs will be avalible
+  if(input ==='sad'|| input === 'depressed'|| input === 'blue') {
```

Comment:
Such a good job using the "or", smart way of not having to add a thousand songs later on!